log/log.go

```diff
@@ -15,12 +15,11 @@ var stdL atomic.Value
 func init() {
 	Replace(zap.NewExample())
 
-	fields := parseFields(os.Getenv("LOG_FIELDS"))
-	if fields == nil {
-		fields = parseFields(os.Getenv("LOG_TAGS"))
-	}
+	name := rucksack.Env("LOG_NAME", "APP_NAME")
+	level := rucksack.Env("LOG_LEVEL")
+	fields := rucksack.Fields(rucksack.Env("LOG_TAGS", "APP_TAGS"))
```

Comment:
same, instead of `MET_TAGS` and `LOG_TAGS`, you can just use `APP_TAGS` *as long* as they are formatted as `key1:val1,key2:val2`, etc